### ISSUE TITLE
Support for openjdk:8-jre-slim

### ISSUE THREAD

**Issue by thanhy** [itsaur/zookeeper-docker-swarm#10]
zookeeper official image is based on openjdk:8-jre-slim -- not alpine!  This repo can't be build locally.   Is there a plan to support debian based image?

**Comment by chatsatourian** [comment]
You are correct @thanhy, zookeeper:3.5 image is based on openjdk:8-jre-slim which is based on debian:stretch-slim. I will make the necessary corrections in order to support the upstream image.
